gems/pending/Gemfile

```diff
@@ -24,7 +24,7 @@ gem "fog",                     "~>2.0.0.pre.0",     :require => false
 gem "fog-core",                "!=1.31.1",          :require => false
 gem "httpclient",              "~>2.5.3",           :require => false
 gem "kubeclient",              "=0.8.0",            :require => false
-gem "hawkular-client",                    :require => false, :git => "git@github.com:hawkular/hawkular-client-ruby.git", :branch => "hawkular-all"
+gem "hawkular-client",                    :require => false, :git => "git@github.com:hawkular/hawkular-client-ruby.git", :branch => "master"
```

Comment:
On 14 Dec 2015, at 11:44, abonas wrote:
> tiny nit: afaik, for master branch you don't have to specify it.
> btw did you already release the gem? if yes, it can be then pulled 
> instead from rubygems instead of the repo here

I only merged to master but did not release yet.
Could be a good idea to do it and then continue from
there.
I wanted to wait on feedback from Federico